sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSessionManagerTest.java

```diff
@@ -411,6 +412,57 @@ void multipleReceiveUnnamedSession() {
         assertFalse(actualLinksCreated.get(0).equalsIgnoreCase(actualLinksCreated.get(1)));
     }
 
+    /**
+     * Validate that session-id specific session receiver is removed after {@link AmqpRetryOptions#getTryTimeout()} is passed.
+     */
+    @Test
+    void singleUnnamedSessionCleanupAfterTimeout() {
+        // Arrange
+        Duration shortTimeout =  Duration.ofSeconds(15);
```

Comment:
Do we need this short timeout? We already have a TIMEOUT constant defined that can be reused. You can bump the seconds up if you need.
https://github.com/Azure/azure-sdk-for-java/blob/0902c492de42ed25164e22fc55ec388041ca12df/sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSessionManagerTest.java#L69